slayTheSpire.js

```diff
@@ -150,9 +153,9 @@ class Zombie extends Monster {
       name: "Zombie",
       icon: "🧟‍♂️ ",
       hp: randInRange(7, 15),
-      energy: 1,
-      hand: 1,
-      deck: [new Shield(), new Strike()]
+      energy: 2,
+      hand: 2,
+      deck: [new Shield(), new Strike(), new Flex()]
```

Comment:
We need to fix that thing where they can use the same card twice in a turn. I'll make an issue…